ui/app/AppLayouts/Wallet/AssetsTab.qml

```diff
@@ -79,13 +81,22 @@ Item {
         }
     }
 
-    ListView {
-        id: assetListView
-        spacing: Style.current.halfPadding
+    ScrollView {
         anchors.fill: parent
-//        model: exampleModel
-        model: walletModel.assets
-        delegate: assetViewDelegate
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
+        ScrollBar.vertical.policy: assetListView.contentHeight > assetListView.height ? ScrollBar.AlwaysOn : ScrollBar.AlwaysOff
```

Comment:
From what I understand, this is done automatically if you don't specify the vertical policy

Reply:
Yes, but it will appear only on hover, while with this code it will display the scrollbar always

Reply:
Aaaah, that make sense